src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -42,10 +42,8 @@ function ProviderSelectionField({
     0,
     providersListLength,
   );
-  const providersListEmpty = currentlyShownProvidersList?.length === 0;
   const loadingProviders =
-    requestStatus === FETCH_STATUS.loading ||
-    requestStatus === FETCH_STATUS.notStarted;
```

Comment:
I think we're using similar logic elsewhere for our loaders so we may want to do a scan for others.

Reply:
I think it's mostly ok in other places, it's more of an issue where we have two different sequential loading states to manage. I can't think of where else we do that, generally we have groups of calls behind one loading state, I think.